backend/src/Command/Content/AbstractContentManagementCommand.php

```diff
@@ -147,6 +151,16 @@ protected function askFieldByDefinition(SymfonyStyle $io, $fieldDefinition, int|
             $q->setMultiline(true);
         }
 
-        return $io->askQuestion($q);
+        $answer = $io->askQuestion($q);
+
+        if (
+            isset($fieldDefinition['type'])
+            && $fieldDefinition['type'] === AgentFieldType::INT
+            && is_numeric($answer)
+        ) {
```

Comment:
Conversion to integer (`(int)$answer`) should check if the `$answer` is definitely numeric beforehand. The current condition (`is_numeric($answer)`) is correct but consider adding a more informative message if the conversion fails or the input isn't a number.